### PR TITLE
Excluded some CI checks for changes in the tools

### PR DESCRIPTION
**Pull request by AlexKoff88**
### Details:
 - Excluded some CI checks for contributions to the tools to speed up the CI and contribution process.

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: .ci/azure/windows.yml [hunk 1]
```diff
@@ -25,6 +26,7 @@ pr:
     - '*.md'
     - '*/layer_tests_summary/*'
     - '*/conformance/*'
+    - 'tools/*'
```
Comment (rkazants) [inline .ci/azure/windows.yml:30]: Here we have a test trying to install openvino-dev. I think we need to exclude MO at least here.
Can you do it please?

Comment (ilya-lavrenov) [reply .ci/azure/windows.yml:30]: I suppose we install openvino-dev here, but don't run any tests.

Comment (rkazants) [reply .ci/azure/windows.yml:30]: Yes correct, but I think it is better to check that `openvino-dev` is installable on Windows. May be we make some requirements changes that affect it.
How do you think?

Comment (AlexKoff88) [reply .ci/azure/windows.yml]: I agree. Reverted.

**Review by rkazants — APPROVED** [review]

**Review by Wovchena — APPROVED** [review]

**Pull request merged**